gordo_components/workflow/workflow_generator/resources/argo-workflow.yml.template

```diff
@@ -1322,12 +1310,12 @@ spec:
             ]
           dependencies:
             - watchman
-            - ml-server
+            - gordo-server
         - name: model-notification-{{ machine.name }}
           template: gordo-notification-svc
           arguments:
             parameters: [{name: model-name, value: "{{ machine.name }}"},
-                         {name: host-name, value: "ml-server-{{ project_name }}"}
+                         {name: host-name, value: "gordo-svr-{{ project_name }}"}
```

Comment:
`gordo-svr` or `gordo-srv` ? Is one obviosly better? Vote-time @ryanjdillon  @milesgranger

Reply:
I would prefer `srv` over `svr` I think.

Reply:
Did a quick google search before going with svr, it seems like it's leaning towards SVR. According to opinions and a shady acronym website.

Reply:
I have no preference.

Reply:
Next question.

Reply:
Then ryan and me vote for `srv`, and since @EinarJohnsen  is only one vote, we win!!!!

Reply:
Changed to srv :new_moon_with_face: